docs/cvl/expr.md

```diff
@@ -29,8 +29,7 @@ expr ::= literal
        | expr "in" id
 
 function_call ::=
-       | [ "invoke" | "sinvoke" ]
-         [ id "." ] id
+       | [ id "." ] id
```

Comment:
```suggestion
       | ( [ id "." ] id | cast_function )
```

Reply:
This isn't correct grammar anyway, you can't do `to_uint256@withrevert(...)` which your change would suggest. Like, so much of this grammar is busted anyway, I'm not willing to hold up this PR, rushing a mistake in after approval.